### ISSUE TITLE
Stripe rejects some creators as “high risk”

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#1755]
Someone who wants to collect donations through Liberapay received the following email from Stripe:

> Thank you for completing our verification process. Unfortunately, we still
> won't be able to accept payments for *redacted*.wordpress.com moving
> forward.
> 
> Stripe can only support users with a low risk of customer disputes—after
> reviewing your submitted information and website, it does seem like your
> business presents a higher level of risk than we can currently support.
> 
> We're sorry that we can't offer ongoing service, and we wish you the best
> of luck with your business.

**Comment by Changaco** [comment]
Stripe is currently kicking out or scaring away some creators:

- https://pub.mastodon.sleeping.town/@unascribed/112513908012567841
- https://ko-fi.com/post/URGENT-PLEASE-READ-IF-YOU-PAY-WITH-STRIPE-P5P7YGWWC

allegedly based on this section of its [Prohibited and Restricted Businesses](https://stripe.com/legal/restricted-businesses) page:

> **Content creation**
>
> - Transactions that provide compensation to creators without an underlying piece of digital content associated. Examples include subscriptions to free, public content or a tip button on a profile page

which, looking at [archived versions of that page](https://web.archive.org/web/*/https://stripe.com/legal/restricted-businesses), was added in January 2024.

See also:

- https://mk.asie.pl/notes/9tsu4ufp3j
- https://x.com/stripesupport/status/1795168327620510004

**Comment by Changaco** [comment]
As long as Stripe doesn't decide that Liberapay itself is unacceptable, the potential solution to these problems remains the implementation of #2238 and/or #2324.

**Comment by Changaco** [comment]
Stripe's policy was updated again in June, removing “Content creation” from the list of prohibited businesses.

**Comment by Changaco** [comment]
After targeting “content creation”, Stripe moved on to targeting “crowdfunding”. I've received reports of users who got their accounts shut down. I've also noticed that “crowdfunding” and “donations” appear in a short list of prohibited use cases that Stripe has added specifically for [cross-border payouts](https://docs.stripe.com/connect/cross-border-payouts). This means that even if Liberapay had a [subsidiary in the US](https://github.com/liberapay/liberapay.org/issues/45), we might not be allowed to use it for cross-border payouts.

**Comment by henry-fisher** [comment]
Yep can confirm we were hit with the crowdfunding shut down on one of our accounts. Appeal was rejected. Hope there's a longer term solution to this!

**Comment by satonotdead** [comment]
Should have used Monero.

**Comment by satonotdead** [comment]
Sorry to hear it, you should move to Monero ASAP.
